to-do-app/src/components/ListView/style.css

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,700;1,100&display=swap');
+
+.active-list-header {
+  margin-top: 46px;
+  margin-bottom: 46px;
+  margin-left: 38px;
+  width: 183px;
+  font-family: 'Roboto', sans-serif;
```

Comment:
I think if you set this on the body, you won't have to add it to all of your selectors: 
```
body {
   font-family: 'Some Font'
}
```

Reply:
I set this in main css file `index.css`, but I've noticed that if I need to make font bold or lighter I need to import exact style for [Roboto](https://fonts.google.com/specimen/Roboto?query=roboto) font. This why in some places I specify this again.

Reply:
Hmmm ... not necessary for this PR, but I would try to think of a way to standardize font weight and family so that you don't have to re-implement in your selectors

Reply:
Okay, I will. Thanks!